src/mutation.cpp

```diff
@@ -518,6 +518,12 @@ void player::activate_mutation( const trait_id &mut )
         print_health();
         tdata.powered = false;
         return;
+    } else if ( !mdata.spawn_item.empty() ) {
+        item tmpitem( mdata.spawn_item.c_str() );
```

Comment:
If WEB_ROPE and VINES3 were to be migrated, would it be acceptable to lose the unique descriptions and replace with the generic `You create a %s.`?